test/e2e/specs/fakerp/admin.go

```diff
@@ -112,4 +126,48 @@ var _ = Describe("Openshift on Azure admin e2e tests [AzureClusterReader]", func
 			Expect(strings.HasPrefix(format, registryPrefix)).To(BeTrue())
 		}
 	})
+
+	It("should ensure no unnecessary VM rotations occured", func() {
```

Comment:
move this into `test/e2e/specs/scaleupdown.go` spec after #795 merges or add it to its own  spec in `test/e2e/specs/reentrant_updates.go`

Reply:
Not sure it has to be its own suite since its a simple read-only test.

Reply:
And this test should be blocking the merge queue.